src/hero.tsx

```diff
@@ -0,0 +1,192 @@
+};
+
+const Hero = React.forwardRef<HTMLDivElement, HeroProps>(
+    (
+        {height, background = 'default', media, desktopMediaPosition = 'left', dataAttributes, ...rest},
+        ref
+    ) => {
+        const {isTabletOrSmaller} = useScreenSize();
+        const isInsideSlideShow = useIsInsideSlideshowContext();
+        const isInverse = background === 'brand' || background === 'brand-secondary';
+
+        if (isTabletOrSmaller) {
+            return (
+                <DisableBorderRadiusProvider>
+                    <div
+                        {...getPrefixedDataAttributes(dataAttributes)}
+                        ref={ref}
+                        style={{
+                            backgroundColor: BACKGROUND_COLOR[background],
+                            ...(height === '100vh' ? {maxHeight: '-webkit-fill-available'} : {}), // Hack to avoid issues in Safari with 100vh
+                            ...assignInlineVars({
+                                [styles.vars.height]: height ?? '',
+                            }),
+                        }}
+                        className={classnames(styles.container, styles.containerMobile)}
+                    >
+                        {media}
+                        <ResponsiveLayout isInverse={isInverse}>
+                            <Box paddingTop={24} paddingBottom={isInsideSlideShow ? 48 : 24}>
+                                <HeroContent {...rest} />
+                            </Box>
+                        </ResponsiveLayout>
+                    </div>
+                </DisableBorderRadiusProvider>
+            );
+        }
+
+        const left =
+            desktopMediaPosition === 'left' ? (
+                media
+            ) : (
+                <Box paddingRight={24}>
+                    <HeroContent {...rest} />
+                </Box>
+            );
+
+        const right =
+            desktopMediaPosition === 'right' ? (
+                media
+            ) : (
+                <Box paddingLeft={24}>
+                    <HeroContent {...rest} />
+                </Box>
+            );
+
+        return (
+            <div
+                ref={ref}
+                style={{
+                    backgroundColor: BACKGROUND_COLOR[background],
+                    ...(height === '100vh' ? {maxHeight: '-webkit-fill-available'} : {}), // Hack to avoid issues in Safari with 100vh
+                    ...assignInlineVars({
+                        [styles.vars.height]: height ?? '',
+                    }),
+                }}
+            >
+                <ResponsiveLayout
+                    isInverse={isInverse}
+                    dataAttributes={{'component-name': 'Hero', ...dataAttributes}}
+                >
+                    <GridLayout
+                        template="6+6"
+                        left={
+                            <Box
+                                paddingY={isInsideSlideShow ? 80 : 56}
```

Comment:
this 80px is a fix for let the Slideshow bullets work with Hero

Reply:
They still work when the padding is 56 in desktop due to image / actions position, we should keep that measure instead of 80px since there's no possible collision.

Images with 56px of padding:

![Screenshot 2023-02-22 at 09 46 01](https://user-images.githubusercontent.com/44420072/220568509-50d356b6-1f47-44f5-a3ac-e63d0f3baeea.png)
![Screenshot 2023-02-22 at 09 45 56](https://user-images.githubusercontent.com/44420072/220568560-93f0e875-07d5-416c-b71f-3affef07a729.png)